app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] == "cp":
```

Comment:
The condition here is incorrect. The function should raise a `ValueError` if the command does not start with 'cp'. Change the condition to `if command_parts[0] != 'cp':`.